docs/genai/api/python.md

```diff
@@ -267,12 +271,27 @@ onnxruntime_genai.Generator.compute_logits()
 
 ### Get output
 
-Returns the output logits of the model.
+Returns an output of the model.
 
 ```python
-onnxruntime_genai.Generator.get_output()
+onnxruntime_genai.Generator.get_output(str: name) -> numpy.ndarray[int32]
```

Comment:
the output can be other data type in addtion to int32

Reply:
the types is determined by the output type of model. I think you can just remove the [int32].

Reply:
and also the [int32] in the line 277.